### PR TITLE
Replicating the schema extract logic for attr_stripped schemas

### PR DESCRIPTION
**Pull request by GGabi**
Essentially a duplicate of: https://github.com/DataIntellectTech/TorQ/pull/659

Adds a test to ensure existing functionality works as intended for the SegmentedTickerplant as that's the only component that uses the attribute stripped schemas.

### CODE REVIEW

File: tests/schemas/insert_data.q
Comment (Papamaci444) [inline tests/schemas/insert_data.q:1]: BTW I think I haven't seen a single file defined in TorQ that uses underscore notation!

Comment (GGabi) [reply tests/schemas/insert_data.q:1]: Oh, I can change that if it's a style issue?

**Review by Papamaci444 — APPROVED** [review]

File: tests/schemas/insert_data.q
Comment (jonathonmcmurray) [reply tests/schemas/insert_data.q:1]: not super important for a filename in tests, but yeah, we avoid underscores generally

File: code/common/pubsub.q [hunk 1]
```diff
@@ -104,17 +104,22 @@ closesub:{[h]
   delhandlef[;h]each t;
  };
 
+/ Scope value to inside the function so we can iterate over a set of tables without loading
+/ them all into memory at once
+/ Takes a symbol to a global table
+extractschema: {t: value x; $[.Q.qp t; t; 0#t]};
```
Comment (jonathonmcmurray) [inline code/common/pubsub.q:110]:
```suggestion
extractschema:{t:value x; $[.Q.qp t; t; 0#t]};
```

mostly a personal preference stylistic thing, but we generally don't have spaces after `:` anywhere in torq

**Review by jonathonmcmurray — APPROVED** [review]
minor comments, but looks good

File: code/common/pubsub.q (hunk 1 above)
Comment (GGabi) [reply code/common/pubsub.q:110]: Got it, I'll make the changes, thanks

**Review by Papamaci444 — APPROVED** [review]
Thanks for fixing these, approved!

**Pull request merged**